api/auth/jwtUtils.test.js

```diff
@@ -6,6 +6,8 @@ const jwt = require('jsonwebtoken');
 const sandbox = sinon.createSandbox();
 const mockVerifier = sandbox.stub();
 
+const JWT_SECRET = 'SOME_SECRET_VALUE_SHHHHH'
```

Comment:
you might want to make this 
`const JWT_SECRET = process.env.JWT_SECRET || 'SOME_SECRET_VALUE_SHHHHH';`
in case they have JWT_SECRET set locally

Reply:
good call.  I made this update.